pkg/checkpoint/process_test.go

```diff
@@ -183,7 +184,7 @@ func TestProcess(t *testing.T) {
 				},
 				"AA": {},
 			},
-			expectStop:        []string{"AA", "kube-system/pod-checkpointer"},
+			expectStart:       []string{"kube-system/pod-checkpointer"},
```

Comment:
Reason for change: if the checkpoint isn't being removed we always start it. In this case it starts in active grace period (because the parent is gone) and moves to remove after the grace period ends.

Reply:
What's the pod-checkpointer's behavior if its manifest is replaced by itself? Would it be restart?

Reply:
nvm, this should be fine.